ui/.build/src/sass.ts

```diff
@@ -166,45 +166,79 @@ async function parseScss(src: string, processed: Set<string>) {
 
 // collect mixable scss color definitions from theme files
 async function parseThemeColorDefs() {
-  const themeFiles = await glob('ui/common/css/theme/_*.scss', { absolute: false });
   const themes: string[] = ['dark'];
+
+  const defaultThemeColors = await loadThemeColorDefs(join(env.themeDir, '_default.scss'));
+  themeColorMap.set('default', defaultThemeColors);
```

Comment:
And to make this work properly, the default theme is now explicitly loaded first.